src/MoBi.Assets/AppConstants.cs

```diff
@@ -495,9 +496,14 @@ public static string RemoveValuePointDescription(double xDisplayValue, double yD
             return string.Format("Remove {3} X: '{0} {5}' Y: '{1} {6}' from {4} '{2}'", xDisplayValue, yDisplayValue, formulaName, ObjectTypes.ValuePoint, ObjectTypes.TableFormula, xDisplayUnit.Name, yDisplayUnit.Name);
          }
 
-         public static string CommitCommandDescription(Module module, IModelCoreSimulation simulation)
+         public static string CommitCommandDescription(IModelCoreSimulation simulation)
          {
-            return $"Commit changes made in Simulation: '{simulation.Name}' to module: '{module.Name}'";
+            return $"Commit changes made in Simulation: '{simulation.Name}'";
+         }
+
+         public static string CommitToModuleCommandDescription(Module module, IModelCoreSimulation simulation, string moduleParent)
+         {
+            return $"Commit changes made in Simulation: '{simulation.Name}' to {moduleParent} module: '{module.Name}'";
```

Comment:
I don't understand the parameter module parent. What does this mean?

Reply:
moduleParent is going to be either "Simulation" or "Project" to highlight where the module lives.